<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSA_Hash</class>
 <widget class="QWidget" name="RSA_Hash">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>682</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>577</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="Functions">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="currentText">
        <string>MD5</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>MD5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RSA</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelN">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>N =3337</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPublicKey">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>public key e =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPrivateKey">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>private key d =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="inputTextEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Historic</family>
         <pointsize>22</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="placeholderText">
        <string>Input</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="input_outputTextEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Historic</family>
         <pointsize>22</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="placeholderText">
        <string>Output/Input</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="outputTextEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Historic</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="Generate">
          <property name="text">
           <string>Generate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Decode">
          <property name="text">
           <string>Decode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Clear">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
